.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,14 +4,12 @@ relates to OR closes: #XXXX
 
 ---
 
-Before we can merge this PR, please make sure that all the following items have been
-checked off. If any of the checklist items are not applicable, please leave them but
-write a little note why.
-
+Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
 - [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
 - [ ] Added **one** label out of the `L-` group to this PR
 - [ ] Added **one or more** labels from the `A-` and `C-` groups to this PR
 - [ ] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
+- [ ] This PR is targeted against the *current*  Milestone ( otherwise discuss if it can be added in time)
```

Comment:
this pr or the linked issue*

Reply:
i think we should sharpen that rule to *this PR*
Every PR that gets merged should have a milestone, i don't want to have to search through the issues to find

Reply:
Then the board will be filled with duplicates, that was the reason we don't add it to PRs.
Where are you looking for RPs anyway ?

Reply:
Check the board, we have it split between issues and PRs now!

Reply:
> Where are you looking for RPs anyway ?

e.g. Clicking into the merge commit from a `git blame`

Reply:
Actually nvm, that's not the case.

Reply:
But still I would rather not have these personal preferences how we track work becoming a rule.
Right now i'm tagging issues where the milestone is missing and linking PRs as best as I can can so I can see everything in the issues board.

Reply:
This is an attempt at standardization.

We've seen tons of inconsistencies with how we do these and asking every PR to be milestoned is a simple rule that will need no exceptions or deep thought to stick to and produces a consistent look on the Projects board.

If you can articulate a case where adopting this rule would make the current situation *worse*, I'm willing leave it to *personal preference*.

Reply:
Having many boards to track is not convenient.
If you're looking for a team wide thing, I'd prefer to return to our previous standard way of tracking issues with linked prs in a single board.

Reply:
why are you up lol

> If you're looking for a team wide thing, I'd prefer to return to our previous standard way of tracking issues with linked prs in a single board.

The problem with that as you know is that there's no way of linking PRs to issues without closing them, making Issues that have multiple PRs like https://github.com/Manta-Network/Manta/issues/669 impossible to use

Reply:
it's not impossible , reopening issues is definitely less inconvenient than having to always track two issues. 
Anyway i'm just going to merge this, and I'll make a new PR for my preference as well.

Reply:
i think a way to have long-running issues work fine is by just rolling them from milestone to milestone
That way they'll always show up in the current issues board

I agree impossible was too strongly worded 😅